UefiPayloadPkg/UefiPayloadEntry/RiscV64/DxeLoadFuncFit.c

```diff
@@ -1,8 +1,7 @@
 /** @file
-  x64-specifc functionality for DxeLoad.
-
-Copyright (c) 2006 - 2020, Intel Corporation. All rights reserved.<BR>
-SPDX-License-Identifier: BSD-2-Clause-Patent
+  RISC-V specific functionality for DxeLoad.
+  Copyright (c) 2023, Rivos Inc

```

Comment:
Please follow the copyright format as in https://tianocore-docs.github.io/edk2-CCodingStandardsSpecification/draft/5_source_files/52_spacing.html#52-spacing.